app/views/foods/index.html.erb

```diff
@@ -0,0 +1,25 @@
+
+<p style="color: green"><%= notice %></p>
+
+<h1>Foods</h1>
+
+<%= link_to "New food", new_food_path %>
+
+<div id="foods">
+  <table class="food-table">
+  <tr>
+    <th>Food</th>
+    <th>Measurement Unit</th>
+    <th>Unit Price</th>
+    <th>Actions</th>
+  </tr>
+  
+  <% @foods.each do |food| %>
+    <%= render food %>
```

Comment:
- [ ] As mentioned by the previous code reviewer, let us try to match the given wireframe. Here is a screenshot showing the differences
![Screenshot from 2023-08-12 23-28-31](https://github.com/OnlyUnknown/Recipe-app-ROR/assets/46388113/4f3efb16-7c30-4fbb-8b60-7f6d0efe3d8d)
![Screenshot from 2023-08-12 23-27-59](https://github.com/OnlyUnknown/Recipe-app-ROR/assets/46388113/08c1a7ce-83df-40d5-b2d0-093b6ea0efe0)
The vertical line and the position of the add food button should be fixed
- [ ] Also let us make the food item clickable so that when clicked we will be redirected to the food details page

Reply:
It matches the wireframe, but as you can see it just made by our style, as they mention in the project requirement
You can see  the columns is food , measurement unit, price and action

Reply:
![Screenshot 2023-08-13 030807](https://github.com/OnlyUnknown/Recipe-app-ROR/assets/116448416/40377863-6988-431a-b60b-ad39fb06f8b5)
and also as you can see, there is bars between them, they are just have even rows to make them look better

Reply:
Hello @addod19 . Kindly review o=the changes we have made.